lib/oli_web/live/common/custom_labels_form.ex

```diff
@@ -0,0 +1,24 @@
+defmodule OliWeb.Common.CustomLabelsForm do
+  use Surface.Component
+  alias Surface.Components.Form
+  alias Surface.Components.Form.{Field, TextInput, Label}
+
+  prop labels, :map, default: %{unit: "unit", module: "module", section: "section"}
```

Comment:
Are these the correct defaults? Shouldn't these values be capitalized?

Reply:
Done